Source/Core/DolphinQt2/Config/SettingsWindow.cpp

```diff
@@ -36,9 +36,6 @@ SettingsWindow::SettingsWindow(QWidget* parent) : QDialog(parent)
     // Actual Settings UI
     SetupSettingsWidget();
 
-    MakeUnfinishedWarning();
-
-    content_inner->addWidget(m_warning_group);
     content_inner->addWidget(m_settings_outer);
 
     content->addLayout(content_inner);
```

Comment:
It's true that we don't need `content_inner` anymore, but removing it didn't get rid of the margins (on my computer, at least).